examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<ol><li>Add link here to the chapter form ["<em>Forecasting: Principles &amp; Practice</em>"](https://otexts.com/fpp2/hierarchical.html) about hierarchical time series</li><li>to efficiently store -&gt; to efficiently store them</li><li>Also there is middle-out approach and optimal reconciliation</li><li>sum up appropriately -&gt; they are coherent</li><li>In this section we will examine the reconciliation methods available in etna</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>